### PR TITLE
Update install_verse.sh approach to dummy texlive install

### LINKED ISSUE #883: texlive-dummy not working
### Container image name

rocker/verse:latest

### Container image digest

_No response_

### What operating system related to this question?

Linux

### System information

_No response_

### Question

Line 79 in /scripts/install_verse.sh

   wget "https://travis-bin.yihui.name/texlive-local.deb"

Is failing.  The web site is not in existence. 

Poking around I find https://github.com/yihui/travis-bin   suggests it is archived. 

I'm not clear why this step rather than just

apt-get install texlive

is used?  It mentions a "dummy" texlive, but not clear to me what this means or an alternative.

### LINKED ISSUE #884: https://travis-bin.yihui.name/texlive-local.deb has moved to https://travis-bin.yihui.org/texlive-local.deb
In `install_verse.sh` the line linked below is failing with a 301 response noting that the link has changed.

https://github.com/rocker-org/rocker-versioned2/blob/02cd56d9fb5c414b792c4c80a12adab5b9712d43/scripts/install_verse.sh#L79C11-L79C58

### PR DESCRIPTION
**Pull request by nickforr**
Fixes #883 and fixes #884 

Based on this discussion: https://github.com/yihui/travis-bin/commit/687592b3f2e8e9aff322c2ac168295180e0ac39a#commitcomment-109192334
And this code: https://github.com/b-data/jupyterlab-r-docker-stack/blob/296fe9d348dfc0b645d2ab7130eb7621f475f57c/verse/latest.Dockerfile#L86-L95

### CODE REVIEW

File: scripts/install_verse.sh [hunk 1]
```diff
@@ -76,9 +76,14 @@ apt-get -y autoremove
 ## Add LaTeX, rticles and bookdown support
 ## tinytex recommends a dummy texlive if using tlmgr manually
 if [[ ! -x "/usr/bin/latex" ]]; then
-    wget "https://travis-bin.yihui.name/texlive-local.deb"
+    apt-get install -y --no-install-recommends equivs
```
Comment (eitsupi) [inline scripts/install_verse.sh:79]:
```suggestion
    apt_install equivs
```

File: scripts/install_verse.sh [hunk 2]
```diff
@@ -76,9 +76,14 @@ apt-get -y autoremove
...
+    cd /tmp
+    wget https://github.com/scottkosty/install-tl-ubuntu/raw/master/debian-control-texlive-in.txt
```
Comment (eitsupi) [inline scripts/install_verse.sh:81]: Doesn't this file have to be removed?

**Review by eitsupi — CHANGES_REQUESTED** [review]

**Review by cboettig — CHANGES_REQUESTED** [review]
other than tidying up the wget file this looks good

**Review by eitsupi — APPROVED** [review]
Thanks!

**Pull request merged**